docker-compose-mysql.yml

```diff
@@ -0,0 +1,61 @@
+version: '3.3'
```

Comment:
Let's not go this way. One more docker-compose file adds an additional layer for support in case of changes.
Let's put an additional docker-compose.mysql.yml file which will extends/overrides the original one.
Source: https://docs.docker.com/compose/extends/

Reply:
In that file you have to override:
- container 'db'
- set POSTGRES_* values to empty
- add MYSQL specific variable for container 'app'

Reply:
Agreed, let me make these changes.

Reply:
@farioas support for `extends` was removed in 3.x. https://docs.docker.com/compose/extends/#extending-services. The docs point to this issue: https://github.com/moby/moby/issues/31101. Any ideas how we can tackle this?

Reply:
I'll try the overrides without using the `extend` keyword and let you know how it goes.

Reply:
Yep, sure, overrides is ok

Reply:
I have updated it with the overrides. It works as expected.

Reply:
Awesome, thanks!

Reply:
btw, please put the file in the same folder as main docker-compose.yml